js/login.js

```diff
@@ -0,0 +1,97 @@
+const emailInput = document.getElementById('email')
+const passwordInput = document.getElementById('password')
+const loginButton = document.querySelector('.login-form__button')
+const eyeicon = document.querySelector('.login-form__input--eye-icon')
+
+//Email Validation
+function validateEmail(email) {
+  const regex = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+/;
+  return regex.test(email);
+}
+
+//Password Validation
+function validatePassword(password) {
+  return password.length >= 8;
```

Comment:
비밀번호가 지금은 8글자까지인데 이후에 바뀔 수 있고, 단순한 숫자값이기에 상수를 정의해서 써주시면 더 좋을 거 같아요!
```suggestion
  const MINIMUM_PW_LENGTH = 8
  return password.length >= MINIMUM_PW_LENGTH;
```